src/liquid/components/ld-link/ld-link.css

```diff
@@ -0,0 +1,52 @@
+.ld-link {
+  --ld-link-font-size: 1rem;
+  --ld-link-size-sm: 0.875rem;
+  --ld-link-size-lg: 1.125rem;
+  --ld-typo-link-brand-color: var(--ld-thm-primary);
+
+  position: relative;
+  display: inline-block;
+  color: var(--ld-typo-link-brand-color);
+  font-size: var(--ld-link-font-size);
+  font-weight: 700;
+  text-decoration: none;
+  
+  &--sm {
```

Comment:
I think the link component should inherit its font-size and line-height. This way it would be easiert to integrate with the ld-typo component for instance. So there should be no need for a size prop. Instead just an example which shows how the inheritance works within the ld-typo component.